annotation/functions.scm

```diff
@@ -401,18 +401,43 @@
     )))
 )
 
+;; Find coding Gene for a given protein
+(define-public find-coding-gene
+  (lambda (protein)
+  (cog-outgoing-set (cog-execute! (BindLink
```

Comment:
We can replace `BindLink` with `GetLink` here as we only need to get the pattern instead of creating it.